pallets/nft/src/lib.rs

```diff
@@ -162,9 +203,10 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn claimed_list)]
-	/// claimed vec for claim type NFT class, to guarantee each user claims once
-	// maximal index of claiming user is 2^16 which is more than enough
-	// TODO consider to reduce it to u16 to save storage usage
+	/// Claimed index vec for `Claim(HashByte32)` type NFT class, 
+	/// to guarantee each user claims once.
+	/// maximal index of claiming user is 2^16 which is more than enough
+	/// TODO consider to reduce it to u16 to save storage usage
```

Comment:
```suggestion
```

Reply:
This has been done already and also TODO should not be shown inside doc (starting with ///)